test/e2e/defaultbackend/default_backend.go

```diff
@@ -98,4 +98,35 @@ var _ = framework.IngressNginxDescribe("Default backend", func() {
 			Expect(resp.StatusCode).Should(Equal(test.Status))
 		}
 	})
+	It("enables access logging for default backend", func() {
+		f.UpdateNginxConfigMapData("enable-access-log-for-default-backend", "true")
+		host := "foo"
+		resp, _, errs := gorequest.New().
+			Get(f.GetURL(framework.HTTP)+"/somethingOne").
+			Set("Host", host).
+			End()
+
+		Expect(len(errs)).Should(Equal(0))
+		Expect(resp.StatusCode).Should(Equal(http.StatusNotFound))
+
+		logs, err := f.NginxLogs()
+		Expect(err).ToNot(HaveOccurred())
+		Expect(logs).To(ContainSubstring("/somethingOne"))
+	})
+
+	It("disables access logging for default backend", func() {
+		f.UpdateNginxConfigMapData("enable-access-log-for-default-backend", "false")
+		host := "bar"
+		resp, _, errs := gorequest.New().
+			Get(f.GetURL(framework.HTTP)+"/somethingTwo").
+			Set("Host", host).
+			End()
+
+		Expect(len(errs)).Should(Equal(0))
+		Expect(resp.StatusCode).Should(Equal(http.StatusNotFound))
+
+		logs, err := f.NginxLogs()
+		Expect(err).ToNot(HaveOccurred())
+		Expect(logs).ToNot(ContainSubstring("/somethingTwo"))
```

Comment:
This i replace this same

Reply:
Why was this line printed?
Is this a debug? I do not remember such lines in version 0.22.0
`127.0.0.1 - [127.0.0.1] - - [25/Feb/2019:21:52:58 +0000] "GET /somethingTwo HTTP/1.1" 404 153 "-" "Go-http-client/1.1" 345 0.000 [internal] - - - - f31a446e7c617a1425739aa8adfa6c67`

can you help me?

Reply:
@arturxx8 I think this is coming from the "upstream server": https://github.com/kubernetes/ingress-nginx/blob/c50fe1bc68e822cab6bc1ae89c4f0f9f4e9b9bfe/rootfs/etc/nginx/template/nginx.tmpl#L612. That's the upstream used when there's no custom default backend configured.

I'm okay to solve this by adding `access_log off;` in the server block.